libfabric.spec.in

```diff
@@ -8,10 +8,15 @@ Url: http://www.github.com/ofiwg/libfabric
 Source: http://www.openfabrics.org/downloads/fabrics/%{name}-%{version}.tar.bz2
 Prefix: ${_prefix}
 
-BuildRequires: libnl-devel
+%if 0%{?configopts:1}
+# skip BuildRequires for custom build
```

Comment:
Is there a way to handle enable/disable BuildRequires on a per component basis, so that we don't toss all build requirements just to disable one provider?